k8s/node.go

```diff
@@ -22,7 +27,16 @@ func GetNodeList(k8sClient *kubernetes.Clientset) (*v1.NodeList, error) {
 }
 
 func GetNode(k8sClient *kubernetes.Clientset, nodeName string) (*v1.Node, error) {
-	return k8sClient.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})
```

Comment:
use Get(nodeName, metav1.GetOptions{}) method is More efficient， why use for loop to get this?  @galal-hussein